app/src/main/java/com/orange/ods/app/ui/AppBarState.kt

```diff
@@ -51,7 +51,7 @@ interface AppBarManager {
  */
 class AppBarState(
     private val navigationState: AppNavigationState,
-    private val searchText: MutableState<TextFieldValue>,
+    val searchText: MutableState<TextFieldValue>,
```

Comment:
We now have both `searchText` and `searchedText` public properties. We can either add a setter for `searchedText` and keep this one private, or remove `searchedText` and use the state directly.